CRM/Case/BAO/Query.php

```diff
@@ -235,7 +229,6 @@ public static function where(&$query) {
         if ($query->_mode == CRM_Contact_BAO_Query::MODE_CONTACTS) {
           $query->_useDistinct = TRUE;
         }
-        $grouping = $query->_params[$id][3];
```

Comment:
It's an unused variable. I think I removed the same line in master in a sibling search class - or I might not have committed that - but I think it's a child of copy & paste